stereovision/ui_utils.py

```diff
@@ -136,8 +136,9 @@ def _set_value(self, parameter, new_value):
         try:
             self.block_matcher.__setattr__(parameter, new_value)
         except BadBlockMatcherArgumentError:
+            print('Bad Block')
             return
-        self.update_disparity_map()
+        #self.update_disparity_map()
```

Comment:
Why is this dropped here? The advantage of updating the disparity map is that when you set a new parameter you can see its effect right away.